eval/stringify.ts

```diff
@@ -0,0 +1,176 @@
+import { isArray } from "@core/unknownutil/is/array";
+import { isBoolean } from "@core/unknownutil/is/boolean";
+import { isFunction } from "@core/unknownutil/is/function";
+import { isInstanceOf } from "@core/unknownutil/is/instance-of";
+import { isNullish } from "@core/unknownutil/is/nullish";
+import { isNumber } from "@core/unknownutil/is/number";
+import { isRecord } from "@core/unknownutil/is/record";
+import { isString } from "@core/unknownutil/is/string";
+import { isSymbol } from "@core/unknownutil/is/symbol";
+import { isUndefined } from "@core/unknownutil/is/undefined";
+import {
+  isVimEvaluatable,
+  type VimEvaluatable,
+  vimExpressionOf,
+} from "./vim_evaluatable.ts";
+
+// Note: Imports only types and is used only in tsdoc.
+// deno-lint-ignore no-unused-vars
+import type { Expression } from "./expression.ts";
+// deno-lint-ignore no-unused-vars
+import type { RawString } from "./string.ts";
```

Comment:
I couldn't find `Expression` nor `RawString` in tsdoc example code. Do we still need to import them?

Reply:
Used in tsdoc as `{@linkcode Expression }`, this will generate valid links in the jsr.io documentation.

Reply:
> And in LSP you can hover over FooBar within {@linkcode FooBar}.

Could you explain this bit more? I don't think it's good idea to add unused import for docs because it is JSR's business but if this unused import affect some usability on LSP, I'll consider (but I couldn't reproduce what you explained with coc.nvim)

Reply:
In vim-lsp:
![vim-lsp-with-@linkcode](https://github.com/user-attachments/assets/414a71cf-15c8-41fe-b7e7-bc21abfbdad0)

Reply:
So the LSP things are for developer of this library rather than users? If so, I'd request you to remove that import and use `{@link URL | label}` instead.